tests/ice40/latches.ys

```diff
@@ -0,0 +1,6 @@
+synth_ice40
+#equiv_opt -assert -map +/ice40/cells_sim.v synth_ice40
```

Comment:
Looks like `equiv_*` does not support latches, either, so perhaps we should resort to simulation for these. Take a look at `tests/simple` for an approach that auto generates a testbench.

Reply:
done